Tests/MSFT_xFirewall.Tests.ps1

```diff
@@ -0,0 +1,15 @@
+Remove-Module -Name MSFT_xFirewall -Force -ErrorAction SilentlyContinue
+Import-Module -Name $PSScriptRoot\..\DSCResources\MSFT_xFirewall -Force -DisableNameChecking
+
+if (! (Get-Module xDSCResourceDesigner))
+{
+    Import-Module -Name xDSCResourceDesigner -ErrorAction SilentlyContinue
+}
+
+Describe 'Schema Validation for MSFT_xFirewall' {
+    Copy-Item -Path ((get-item .).parent.FullName) -Destination $(Join-Path -Path $env:ProgramFiles -ChildPath 'WindowsPowerShell\Modules\') -Force -Recurse
+    $result = Test-xDscResource MSFT_xFirewall
+    It 'should pass Test-xDscResource' {
+        $result | Should Be $true
+    }
+}
```

Comment:
Please end a file with a newline [See item 5 in style guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md)